src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage.cs

```diff
@@ -88,14 +88,20 @@ internal partial class SQLitePersistentStorage : AbstractPersistentStorage
         }
 
         public override void Dispose()
+        {
+            var task = DisposeAsync().AsTask();
+            task.Wait();
```

Comment:
this 'Wait' is equivalent to how we disposed before.  Specifically, we would get a flush task and then do a blocking .Wait on that.  This just moves that wait here.  If someone goes through teh DIsposeAsync path, they won't hit this.

Reply:
agree